CPP needs to be set since it is used by AM_CHECK_PYTHON_HEADERS.
--- configure.ac.orig	2019-06-16 04:14:20.000000000 +1000
+++ configure.ac	2021-03-16 20:21:17.000000000 +1100
@@ -47,6 +47,7 @@
 
 # Checks for programs.
 AC_PROG_CC
+AC_PROG_CPP
 AM_PROG_CC_C_O
 AC_PROG_MKDIR_P
 
